src/main/java/com/avairebot/requests/service/PunService.java

```diff
@@ -0,0 +1,82 @@
+package com.avairebot.requests.service;
+
+import java.util.List;
+
+public class PunService
+{
+    private List<Pun> results;
+
+    private int page;
+
+    private int limit;
+
+    public int getNext_page() {
```

Comment:
All the getter methods should all use camel case lettering, this also includes all the methods below in this class, it's just nicer to have everything be a bit more consistent.

Reply:
The camel case lettering should now be set.